BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewModel.swift

```diff
@@ -19,23 +19,20 @@ public struct BottomSheetViewModel {
     public var shouldDismissForTapOutside: Bool
     public var shadowOpacity: Float
     public var closeButtonA11yLabel: String
-    public var closeButtonA11yIdentifier: String
 
     public init(
         cornerRadius: CGFloat = BottomSheetViewModel.UX.cornerRadius,
         animationTransitionDuration: TimeInterval = BottomSheetViewModel.UX.animationTransitionDuration,
         backgroundColor: UIColor = .clear,
         shouldDismissForTapOutside: Bool = true,
         shadowOpacity: Float = BottomSheetViewModel.UX.shadowOpacity,
-        closeButtonA11yLabel: String,
-        closeButtonA11yIdentifier: String
+        closeButtonA11yLabel: String
     ) {
         self.cornerRadius = cornerRadius
         self.animationTransitionDuration = animationTransitionDuration
         self.backgroundColor = backgroundColor
         self.shouldDismissForTapOutside = shouldDismissForTapOutside
         self.shadowOpacity = shadowOpacity
         self.closeButtonA11yLabel = closeButtonA11yLabel
-        self.closeButtonA11yIdentifier = closeButtonA11yIdentifier
```

Comment:
Pretty sure this should be corrected to be used instead of cleaned up

Reply:
Fixed passing correct a11yIdentifier in `BottomSheetViewController.swift`